README.md

```diff
@@ -2,9 +2,8 @@
 
 This repository contains the documentation source files for [coder.com/docs](https://coder.com/docs).
 
-Pre-production documentation is drafted in a private repository before being
-synchronized with this repository as releases are made public.
+Is something confusing, unclear, or just want to contribute something new? [Open an issue or pull request](https://github.com/cdr/docs/issues/new)! While you can't currently build the docs site locally, you'll be able to preview your changes in the context of the site through a deploy preview. 
```

Comment:
these lines are a bit long, would recommend a space at 80 chars to follow markdown convention